elastic4s-core/src/main/scala/com/sksamuel/elastic4s/api/PipelineAggregationApi.scala

```diff
@@ -42,8 +42,8 @@ trait PipelineAggregationApi {
   def minBucketAggregation(name: String, bucketsPath: String): MinBucketPipelineAgg =
     MinBucketPipelineAgg(name, bucketsPath)
 
-  def movingAverageAggregation(name: String, bucketsPath: String): MovAvgPipelineAgg =
-    MovAvgPipelineAgg(name, bucketsPath)
+  def movingFunctionAggregation(name: String, bucketsPath: String, script: String, window: Integer): MovFnPipelineAgg =
```

Comment:
was moving average removed ?

Reply:
The moving_avg aggregation has been removed. See the [breaking changes here](https://www.elastic.co/guide/en/elasticsearch/reference/current/migrating-8.0.html#breaking_80_rest_api_changes).